commands.go

```diff
@@ -300,40 +300,23 @@ func unbanCommand(u *user, args []string) {
 }
 
 func helpCommand(u *user, _ []string) {
-	// u.system("Welcome to Devzat! Devzat is chat over SSH: github.com/quackduck/devzat")
-	// u.system("Because there's SSH apps on all platforms, even on mobile, you can join from anywhere.")
-	// u.system("")
-	// u.system("Interesting features:")
-	// u.system("• Many, many commands. Run ./commands.")
-	// u.system("• Rooms! Run ./room to see all rooms and use ./room #foo to join a new room.")
-	// u.system("• Markdown support! Tables, headers, italics and everything. Just use \n in place of newlines.")
-	// u.system("• Code syntax highlighting. Use Markdown fences to send code. Run ./example-code to see an example.")
-	// u.system("• Direct messages! Send a quick DM using =user <msg> or stay in DMs by running ./room @user.")
-	// u.system("• Timezone support, use ./tz Continent/City to set your timezone.")
-	// u.system("• Built in Tic Tac Toe and Hangman! Run ./tic or ./hang <word> to start new games.")
-	// u.system("• Emoji replacements! (like on Slack and Discord)")
-
-	// u.system("For replacing newlines, I often use bulkseotools.com/add-remove-line-breaks.php.")
-
-	// u.system("Made by Ishan Goel with feature ideas from friends.")
-	// u.system("Thanks to Caleb Denio for lending his server!")
-	u.system(`Welcome to Devzat! Devzat is chat over SSH: github.com/quackduck/devzat  
-Because there's SSH apps on all platforms, even on mobile, you can join from anywhere.
-
-Interesting features:
-* Many, many commands. Run ./commands.
-* Rooms! Run ./room to see all rooms and use ./room #foo to join a new room.
-* Markdown support! Tables, headers, italics and everything. Just use \\n in place of newlines.
-* Code syntax highlighting. Use Markdown fences to send code. Run ./example-code to see an example.
-* Direct messages! Send a quick DM using =user <msg> or stay in DMs by running ./room @user.
-* Timezone support, use ./tz Continent/City to set your timezone.
-* Built in Tic Tac Toe and Hangman! Run ./tic or ./hang <word> to start new games.
-* Emoji replacements! \:rocket\: => :rocket: (like on Slack and Discord)
-
-For replacing newlines, I often use bulkseotools.com/add-remove-line-breaks.php.
-
-Made by Ishan Goel with feature ideas from friends.  
-Thanks to Caleb Denio for lending his server!`)
+	u.system("Welcome to Devzat! Devzat is chat over SSH: github.com/quackduck/devzat")
```

Comment:
Why revert this?

Reply:
Had some issues with \n not working properly. Make a pr to my branch fixing it or fix it yourself later.